package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "standard",
     "test": "npm run test:node && npm run test:browser",
     "test:browser": "karma start karma.conf.js",
-    "test:node": "tape ./tests/index.js",
+    "test:node": "node tests/",
```

Comment:
No, this didn't work with tape anymore. I now slightly changed the test command calling ``index.js`` directly, though I have to admit that I doesn't really see the advantage, result is the same. Anyway. :-)